.github/actions/e2e_test/action.yml

```diff
@@ -189,7 +189,7 @@ runs:
       id: create-prefix
       shell: bash
       run: |
-        uuid=$(uuidgen)
+        uuid=$(uuidgen | head -c 23)
```

Comment:
That shouldn't be needed, as the next line should cut only the part after the last dash?

Reply:
I was triaging a run and saw the old error output that was already fixed by https://github.com/edgelesssys/constellation/pull/1662